app/src/main/java/com/jerboa/feat/BlurTypes.kt

```diff
@@ -0,0 +1,45 @@
+package com.jerboa.feat
+
+import android.os.Parcelable
+import androidx.annotation.StringRes
+import androidx.compose.ui.res.stringResource
+import com.jerboa.R
+import com.jerboa.datatypes.types.PostView
+import com.jerboa.toEnum
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+enum class BlurTypes(@StringRes val resId: Int) : Parcelable {
+    Nothing(R.string.app_settings_nothing),
+    NSFW(R.string.app_settings_blur_nsfw),
+    NsfwExceptFromNsfwCommunities(R.string.app_settings_blur_nsfw_except_from_nsfw_communities),
```

Comment:
Remove this type, as discussed we keep only, blur nothing, blur nsfw and blur nsfw but not in community view

Reply:
@MV-GH I didn't understand you, only these 3 options remain. 4 option (ExceptFromNsfwInsideCommunity) it technical option. In community activity I swap  NsfwExceptFromNsfwCommunities option for it.

Reply:
That's not how I meant that second approach. The second approach was, turn NSFWExceptInsideCommunity into Nothing if passed to communityActiviity else convert it to NSFW and then you only have to handle those two cases

Reply:
> else convert it to NSFW and then you only have to handle those two cases

I can't figure out how and where to do this. In Main Activity for every Activity?

Reply:
yes but you can do it once under appsetting and pass that. except for community activity where you then pass the other option. But first I have to know if Dessalines would accept such approach.

@dessalines Would you be against the above approach or do you rather that we have a extra param indicating that this is community view. Or do you have a better approach in mind?

Reply:
> NSFWExceptInsideCommunity into Nothing if passed to communityActiviity

Yes that seems fine. No point in adding an extra enum when it can figure that out by context, in the CommunityActivity.

Reply:
@MV-GH Am I understanding correctly that "NsfwExceptFromNsfwCommunities" works as "Nothing" only for nsfw communities, otherwise it works as "NSFW"? If this is true then I can't change it in MainActivity because I need the community type.

Reply:
For what do you need the communityType? You map it there either to Nothing or NSFW ordinals depending if its for CommunityView

Reply:
@MV-GH what behavior should there be with the nsfw posts if Community.nsfw = false and blur option = "NsfwExceptFromNsfwCommunities"?

Reply:
Remove the community nsfw option, we don't care anymore if the post comes from a nsfw community or not. And if the function receives NsfwExceptFromNsfwCommunity then throw with message likr this type should ve already been transformed into another